### PR TITLE
Ticket #2 Frontend/Backend Integration

### PR DESCRIPTION
**Pull request by jolentino**
In this branch I've completed the ticket requirements: connecting both ends and solving frontend bugs.

Authentication should already handled by JWT.

### CODE REVIEW

**Comment by jolentino** [comment]
ticket2
Snackbar bug resolved
Authentication that checks for user in local storage removed since we're authenticating via http only cookie via JWT, hence why the page redirects to dashboard

File: client/src/components/wrapped_components/WrappedFormik.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+							}}
+							InputProps={{ classes: { input: classes.inputs } }}
+							name="username"
+							autoComplete="username"
+							autoFocus
+							helperText={touched.username ? errors.username : ''}
+							error={touched.username && Boolean(errors.username)}
+							value={values.username}
+							onChange={handleChange}
+						/>
```
[the remark this answers is not in the document]
Comment (jolentino) [reply client/src/components/wrapped_components/WrappedFormik.js:58]: Just learned new conditionals, thanks!!

File: client/src/components/wrapped_components/WrappedFormik.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+				action(args).then(
+					() => {
+						history.push('/dashboard');
+						return;
+					},
+					(error) => {
+						setSubmitting(false);
+						setStatus(error);
+						setOpen(true);
+					}
```
Comment (johnsciutto) [inline client/src/components/wrapped_components/WrappedFormik.js:35]: I'm not very familiar with Formik, but doesn't this belong in a `.catch()`?

Comment (jolentino) [reply client/src/components/wrapped_components/WrappedFormik.js:35]: I may certainly refactor that as a .catch() for better readability

**Pull request merged**